### PR TITLE
install tiles

### LINKED ISSUE #2089: Current development releases by lexelby - fix prefs AND smoothing - make probs on Ubuntu 22.10
When an existing design with defined "fill stitches" is loaded I can choose params in the case of "Ripple".
In the case of autofill or contour-fill I recieve an error.
Same error occurs when Trying to parm a new object, for ex. a simple circle or rectangle with a fill colour.
Bildschirmfoto vom 2023-02-23 16-42-32
Bildschirmfoto vom 2023-02-23 10-23-18

### PR DESCRIPTION
**Pull request by lexelby**
fixes #2089 

@bianchifan I think this should fix your "client data" error.

### CODE REVIEW

File: electron/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "pack": "electron-builder --dir",
     "dist": "node .electron-vue/build.js && electron-builder",
-    "dev": "node --openssl-legacy-provider .electron-vue/dev-runner.js"
+    "dev": "node --openssl-legacy-provider .electron-vue/dev-runner.js || node .electron-vue/dev-runner.js"
```
Comment (kaalleen) [inline electron/package.json:10]: This breaks it for me. Do we need to support old node?

Comment (lexelby) [reply electron/package.json:10]: Great, let's get rid of it and see if anyone complains :)

Comment (kaalleen) [reply electron/package.json]: Oh, I thought "old" was the other way around :D . Which version of nodejs do you use?

Comment (lexelby) [reply electron/package.json]: Ohhhh, okay.  I guess I could just upgrade my node then :)

**Review by kaalleen — APPROVED** [review]

**Pull request merged**